### PR TITLE
feat(eslint): add i18n eslint plugin

### PR DESCRIPTION
**Pull request by ismay**
See this slack thread: https://dhis2.slack.com/archives/CPGUFVC56/p1615363540343500

This adds https://github.com/edvardchen/eslint-plugin-i18next, which warns on untranslated strings in jsx. The settings are what I've been using in the scheduler. We could tweak them a bit more later on, and for example add linting of certain attributes as well. I'm ignoring this rule for test files, as react components defined in test files won't be shipped to the end user.

I've set it to warn initially so that it's not a breaking change and to allow us to try it out first. Once we're happy with it we could move it to error and fine-tune the rule.

### CODE REVIEW

**Review by HendrikThePendric — APPROVED** [review]
Adding it like this would be fine with me, but we could also look into perhaps declaring a few attribute names where we do want to enforce `i18n`.

File: config/js/eslint-react.config.js [hunk 1]
```diff
@@ -14,6 +16,10 @@ module.exports = {
     },
 
     rules: {
+        'i18next/no-literal-string': [
+            'warn',
+            { markupOnly: true, onlyAttribute: [''] },
```
[the remark this answers is not in the document]
Comment (ismay) [reply config/js/eslint-react.config.js:21]: Yeah I could do that. Is it necessary to translate displayName though? That's only for developers right?

[the remark this answers is not in the document]
Comment (ismay) [reply config/js/eslint-react.config.js]: I've enabled it for `label` in 4e5acac. I've not enabled it for displayName as that's only used in error messages and not user facing.

[the remark this answers is not in the document]
Comment (ismay) [reply config/js/eslint-react.config.js]: Enabling it for label has the expected result, so seems like a good addition:

Screenshot 2021-03-10 at 11 21 28

**Comment by varl** [comment]
The "warn" aspect relates to #366.

**Pull request closed without merging**